### ISSUE TITLE
Screen reader does not announce it's in a menu for quick fix actions

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#167596]
Testing #166761 on Linux with Ubuntu's default screen reader

1. Open a native or custom context menu (controlled by the title bar style setting)
2. Notice it announces a "menu" when it opens
3. Open the quick fix menu via keybinding or mouse
4. Notice no "menu" is announced

This might be fixed by using an [aria-role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role) or corresponding HTML menu element
